plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -57,9 +57,21 @@ export class KubernetesBuilder {
 
   public async build() {
```

Comment:
Ah interesting, turns out this changes the public API as the return type here is inferred. We'll likely want to add in an explicit return type here in order to keep the API intact.

Reply:
How's this? 3a4fc2b70b